### PR TITLE
2.1.2-beta1 release

### PR DESCRIPTION
**Pull request by outofambit**
- [ ] ✅ finalize release notes
- [ ] 🚢 ship
- [ ] 🤝 merge

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,16 @@
 {
   "releases": {
+    "2.1.2-beta1": [
+      "[Added] Hide whitespace in historical commits - #8045. Thanks @say25!",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Added] Option to hide whitespace in historical commits - #8045. Thanks @say25!",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+      "[Fixed] Tiny images in two-up image rendering - #8103",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Small images are scaled down too much in two-up image rendering - #7520",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+      "[Fixed] Manual conflict resolution for binary files always chose same version - #8059",
```
Comment (billygriffin) [inline changelog.json]: Present tense elsewhere so made this one present as well for consistency
```suggestion
      "[Fixed] Manual conflict resolution for binary files always choose same version - #8059",
```

**Comment by outofambit** [comment]
reopening this with the correct branch and PR name 🤕

**Comment by outofambit** [comment]
see #8131 for the correct numbering and release

**Pull request closed without merging**